Makefile

```diff
@@ -75,17 +96,20 @@ PACKAGE_RELEASE := 1
 
 DOCKERFILE = Dockerfile
 
-seego = docker run --rm -t -v "$(CURDIR):$(CURDIR)" -w "$(CURDIR)" -e "CGO_ENABLED=$$CGO_ENABLED" -e "GOOS=$$GOOS" -e "GOARCH=$$GOARCH" -e "GOARM=$$GOARM" -e "GOMIPS=$$GOMIPS" grafana/agent/seego
-docker-build = docker build $(DOCKER_BUILD_FLAGS)
 
+# seego is used by default when running bare make commands such as `make dist` this uses an image that has all the necessary libraries to cross build
+#	when using drone the docker in docker is more problematic so instead drone uses seego has the base image then calls make running "raw" commands
+seego = docker run --rm -t -v "$(CURDIR):$(CURDIR)" -w "$(CURDIR)" -e "CGO_ENABLED=$$CGO_ENABLED" -e "GOOS=$$GOOS" -e "GOARCH=$$GOARCH" -e "GOARM=$$GOARM" -e "GOMIPS=$$GOMIPS"  grafana/agent/seego
+docker-build = docker build $(DOCKER_BUILD_FLAGS)
 ifeq ($(CROSS_BUILD),true)
 DOCKERFILE = Dockerfile.buildx
-
 docker-build = docker buildx build --push --platform linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7 $(DOCKER_BUILD_FLAGS)
 endif
 
-ifeq ($(BUILD_IN_CONTAINER),false)
-seego = "/seego.sh"
+# If being called from the drone executable this will be set to true, in this case we want the go_wrapper to set the
+#	CC and CCX and other flags for the appropriate architecture
+ifeq ($(DRONE),true)
+	seego = bash ./.drone/go_wrapper.sh 
```

Comment:
Shouldn't `/seego.sh` (or `/go_wrapper.sh`) still exist from Drone since it's running from the seego container?